rust/Cargo.toml

```diff
@@ -31,9 +32,10 @@ serde_json = "1"
 
 [features]
 bench = ["ring"]
-default = ["std"]
+default = ["std", "aesni"]
 staticlib = []
 std = []
+noni = ["aes-soft"]
```

Comment:
Not a big fan of the "noni" name. Among other things

Instead of using an explicit feature for this, how about gating it on the `aes-soft` crate itself? e.g.

```rust
#[cfg(features = "aes-soft")]
```